container/nginx/confs/proxy.tlsconf

```diff
@@ -50,6 +50,21 @@ server {
         return 302 https://strn.network;
     }
 
+    location /ipns/ {
```

Comment:
Shouldn't we rewrite redirections? so once ipfs gateway redirs ipns to ipfs CID, it's a saturn one

Reply:
/ipns endpoints don't return redirections, the gateway resolves the cid internally and returns the content. 

`curl 'https://ipfs.io/ipns/tokens.uniswap.org' -v -o /dev/null -k`